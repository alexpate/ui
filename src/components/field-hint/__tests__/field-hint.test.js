/* @flow */
/* eslint-env jest */
import expect from 'expect';
import {shallow} from 'enzyme';
import {testComponent} from 'utils/test/react';

import FieldHint from '../field-hint';

describe('components/field-hint', () => {
  describe('FieldHint', () => {
    const create = testComponent(FieldHint, () => ({
      // insert your default props here ...
    }));

    it('should render shallow component ok', () => {
      const {element} = create({
        // insert test specific props here ...
        children: 'Child',
      });
      const result = shallow(element);
      expect(result).toExist('should render OK');
    });
  });
});
