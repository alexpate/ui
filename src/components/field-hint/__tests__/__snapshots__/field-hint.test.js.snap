// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldHint should render shallow component ok 1`] = `
<span
  className="e1q9f2xp3 css-1onqzxr"
  style={Object {}}
>
  <span
    className="em7k1990 css-kxwt0l"
    color="navy600"
    onClick={undefined}
    size="extra-small"
    style={Object {}}
  />
</span>
`;
